packages/mason_cli/lib/src/commands/publish.dart

```diff
@@ -52,6 +53,19 @@ class PublishCommand extends MasonCommand {
       return ExitCode.software.code;
     }
 
+    final brickYaml = checkedYamlDecode(
+      brickYamlFile.readAsStringSync(),
+      (m) => BrickYaml.fromJson(m!),
+    );
+
+    if (brickYaml.publishTo == 'none') {
```

Comment:
We should also use the `publishTo` value to determine where the brick is published if it isn't set to `none`.